File: Codeforces/Juicer.java
import java.util.* ;
public class Juicer {
        public static void main(String[] args) {
                Scanner sc = new Scanner(System.in) ;
                int n , big = 0 , waste = 0 ;
                n = sc.nextInt() ; 
                big = sc.nextInt() ; 
                waste = sc.nextInt() ;
                int arr[] = new int [n] ;
                for (int i = 0 ; i < n ; i++) {
                        arr[i] = sc.nextInt() ;
                } 
                int ans = 0 ;
                int w = 0 ;
                for (int i = 0 ; i < n ; i++) {
                        if (arr[i] > big) continue ;
                        else {
                                w += arr[i] ;
                        }
                        if (w > waste) {
                                ans++ ;
                                w = 0 ;
                        }
                }
                System.out.println(ans) ;
        }
}package Competitive-Programming.Codeforces;

public orclass Juicer {
        
}

File: Codeforces/Juicer.kt
import java.util.*
fun main(args: Array<String>) {
val sc = Scanner(System.`in`)
val testcases = sc.nextInt()
for (i in 0 until testcases) {
        val n = sc.nextInt()
        val x = sc.nextInt()
        if (n == 1 || n == 2) {
                println("1")
        } 
        else if ((n - 2) % x == 0) {
                println((n - 2) / x + 1)
        } 
        else if ((n - 2) % x != 0) {
                println((n - 2) / x + 2)
        }
}
        sc.close()
}